src/components/avatar/avatar.bootstrap.scss

```
@@ -0,0 +1,5 @@
+@use '../../styles/utilities' as *;
+
+.rounded {
```

Use the property selector:
```suggestion
:host([shape='rounded']) [part='base'] {
```